.cbmc-batch/jobs/aws_cryptosdk_sign_header/Makefile

```diff
@@ -38,6 +36,9 @@ DEPENDENCIES += $(SRCDIR)/helper-src/openssl/bn_override.goto
 DEPENDENCIES += $(SRCDIR)/helper-src/openssl/ec_override.goto
 DEPENDENCIES += $(SRCDIR)/helper-src/openssl/evp_override.goto
 
-###########
+
```

Comment:
Add a comment explaining why you remove these function bodies

Reply:
Done.